core/src/main/resources/static/js/requestAvroSchemaUpload.js

```diff
@@ -144,6 +144,14 @@ app.controller("requestSchemaCtrl", function($scope, $http, $location, $window)
                         return;
                     }
 
+                    if($scope.addSchema.forceRegister === true) {
+                    if($scope.addSchema.remarks == null) {
+                    $scope.addSchema.remarks = " Force Register Schema option overriding schema compatibility has been selected."
+                    } else {
+                     $scope.addSchema.remarks += " Force Register Schema option overriding schema compatibility has been selected."
```

Comment:
in both the conditions, same remarks are set. we can remove the condition block

Reply:
If it is set to += if they add remarks it will be added to the end.

but if they dont set a remark it will then come out as "undefined Force...." 

so if block is needed to make sure remarks are not null to give best experience to user. Also copy may change Harishini is just checking on it at the mo i think.

Reply:
I meant, if we can define a variable for the remarks string. Currently it is duplicated